### PR TITLE
testnet announcement

### PR DESCRIPTION
**Pull request by johannbarbie**
resolves https://github.com/leapdao/leapdao-website/issues/16

### Delivery

- [x] figure out and create writeup (not for blog) about our [target audiences](https://docs.google.com/document/d/1FsItXOUKhadFxzixiKE7fS5zNcSsATfjq3Wo4Hwv9us)
- [x] what are we launching and why (testing plasma with transfers)
- [x] where to start and call to action (consider splitting by target audience)

**Featuring:**
- [x] contacts: Slack, documentation
- [x] proper twitter image and og:image
- [x] proper meta tags (description, title)

**Style:**
- [x] easy to read, no(or very little) slang
- [x] no grammar errors, no style errors, ideally proof read by native speaker
- [x] actionable (see above)
- [x] preview should look nicely when shared on social networks (facebook, telegram, twitter)

### CODE REVIEW

File: src/blog/content/Plasma-Testnet-Launched.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+### For users:
+- Get some tokens from the [testnet faucet](http://testnet.leapdao.org/faucet).
+- [Deposit and transfer](http://testnet.leapdao.org/wallet) some tokens. Transfers will happen on the Leap Network, free and fast. You can also try out our [mobile wallet alpha](https://github.com/leapdao/mobile-plasma-wallet).
+- Download and [synchronize a Plasma node](https://github.com/leapdao/leap-node) to check the validity of the chain and security of your funds.
+- Keep an eye on [recent governance proposals](http://testnet.leapdao.org/governance) that may change security assumptions.
+- [Exit](http://testnet.leapdao.org/wallet) your tokens back to the root chain.
+
+### For developers:
+- Request your [token to be registered](http://testnet.leapdao.org/registerToken) on the network. The token will receive a “color” and deposits and exits will be enabled for it.
+- Download and [synchronize a Plasma node](https://github.com/leapdao/leap-node), or use our infrastructure at http://node1.testnet.leapdao.org to query the API.
```
Comment (sunify) [inline src/blog/content/Plasma-Testnet-Launched.md]: http://node1.testnet.leapdao.org:8545

File: src/blog/content/Plasma-Testnet-Launched.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- A More Viable Plasma network built on Tendermint.
+- Single Plasma operator run by LeapDAO community.
+- Ability to transfer ERC20/ERC721 tokens.
+- Contract upgrades secured by Minimal Viable Governance.
+The 0.5 Driftwood Release is the first step on the LeapDAO roadmap to launch a public Plasma network supporting provable computations. You can read more about our development proposition in [the whitepaper](https://docs.google.com/document/d/1vStTjqvqZGyiI5AVtpwCIMlHFnzC_4bbixsCfs27-M8).
+
+No matter if you are a cryptocurrency user or a developer, there is something for you in this release:
+
+### For users:
+- Get some tokens from the [testnet faucet](http://testnet.leapdao.org/faucet).
```
Comment (sunify) [inline src/blog/content/Plasma-Testnet-Launched.md]: Does it work?

File: src/blog/content/Plasma-Testnet-Launched.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- Keep an eye on [recent governance proposals](http://testnet.leapdao.org/governance) that may change security assumptions.
+- [Exit](http://testnet.leapdao.org/wallet) your tokens back to the root chain.
+
+### For developers:
+- Request your [token to be registered](http://testnet.leapdao.org/registerToken) on the network. The token will receive a “color” and deposits and exits will be enabled for it.
+- Download and [synchronize a Plasma node](https://github.com/leapdao/leap-node), or use our infrastructure at http://node1.testnet.leapdao.org to query the API.
+- Check out [the documentation](http://docs.leapdao.org) on how to query the chain and send transactions. 
+
+
+If you are curious to know more about the project, read more about us at the [website](https://leapdao.org/) and our [twitter](https://twitter.com/leapdao). If you want to become involved in the project, [join our Slack](https://docs.google.com/forms/d/e/1FAIpQLSd8_wDGDAi__HvfYEWNK_bvJzIkxwHHRVL6AFEfJewBd2Vn9A/viewform) — we are looking for like-minded people (not only developers ;)).
```
Comment (troggy) [inline src/blog/content/Plasma-Testnet-Launched.md]:
```suggestion
If you are curious to know more about the project, read more about us on the [website](https://leapdao.org/) and our [twitter](https://twitter.com/leapdao). If you want to become involved in the project, [join our Slack](https://docs.google.com/forms/d/e/1FAIpQLSd8_wDGDAi__HvfYEWNK_bvJzIkxwHHRVL6AFEfJewBd2Vn9A/viewform) — we are looking for like-minded people (not only developers ;)).
```

File: src/blog/content/Plasma-Testnet-Launched.md (hunk 2 above)
Comment (johannbarbie) [reply src/blog/content/Plasma-Testnet-Launched.md]: 😬

**Pull request merged**